test_outputs/query_execution/function_parameter_count_changed.snap

```diff
@@ -22,6 +24,10 @@ snapshot_kind: text
       "current_parameter_count": Uint64(0),
       "name": String("function_with_parameters_removed"),
       "old_parameter_count": Uint64(2),
+      "old_parameter_names": List([
+        String("_"),
+        String("_"),
+      ]),
```

Comment:
It's unfortunate that the tests don't name these parameters better, so we can determine e.g. whether the order is correct.

Would you mind editing the test crate to give the parameters names like `a, b` etc. so we can ensure we use appropriate order here?